src/cms/templates/languages/language_list_row.html

```diff
@@ -2,33 +2,38 @@
 
 <tr class="border-t border-solid border-gray-200 hover:bg-gray-100">
 	<td class="pl-2">
-		<a href="{% url 'edit_language' language_code=language.code %}" class="block py-3 px-2 text-gray-800">
+		<a href="{% url 'edit_language' language_slug=language.slug %}" class="block py-3 px-2 text-gray-800">
 			{{ language.translated_name }}
 		</a>
 	</td>
 	<td>
-		<a href="{% url 'edit_language' language_code=language.code %}" class="block py-3 px-2 text-gray-800">
+		<a href="{% url 'edit_language' language_slug=language.slug %}" class="block py-3 px-2 text-gray-800">
 			{{ language.native_name }}
 		</a>
 	</td>
 	<td>
-		<a href="{% url 'edit_language' language_code=language.code %}" class="block py-3 px-2 text-gray-800">
-			{{ language.code }}
+		<a href="{% url 'edit_language' language_slug=language.slug %}" class="block py-3 px-2 text-gray-800">
+			{{ language.slug }}
 		</a>
 	</td>
 	<td>
-		<a href="{% url 'edit_language' language_code=language.code %}" class="block py-3 px-2 text-gray-800">
+		<a href="{% url 'edit_language' language_slug=language.bcp47_tag %}" class="block py-3 px-2 text-gray-800">
```

Comment:
```suggestion
		<a href="{% url 'edit_language' language_slug=language.slug %}" class="block py-3 px-2 text-gray-800">
```
The link to the language should use the slug, not the tag. Otherwise, the "create new language" form opens when clicking on the tag.